net/librdkafka/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        edenhill librdkafka 1.7.0 v
+github.setup        edenhill librdkafka 1.8.2 v
 revision            1
```

Comment:
the `revision` field should be set to "zero" when updating the version

Reply:
Done